Model/Config/Survey.php

```diff
@@ -140,6 +144,24 @@ public function startDate($module = null)
         return $this->scopeConfig->getValue($configPath);
     }
 
+    /**
+     * @param $module
+     * @return bool|null
+     */
+    private function resolver($module = null): bool
+    {
+        $isModuleInstall = null;
+        switch ($module):
```

Comment:
Would an IF statement be better here?

Reply:
Maybe but for example if we add more condition it will be not very beautifyl.
A lot of "elseif" -conditions looks bad.